_posts/2024-08-01-contributing-to-che-docs.adoc

```diff
@@ -0,0 +1,51 @@
+---
+title: Contributing to Eclipse Che documentation
+layout: post
+author: Jana Vrbkova
+description: >-
+  Contribute to Eclipse Che documentation using an Eclipse Che workspace.
```

Comment:
```suggestion
  Contribute to Eclipse Che documentation using an Eclipse Che cloud development environment.
```

Reply:
if I understand it correctly, it's better to refer to it as a CDE than a workspace (in general)? I want to make sure I use the correct expression in the future.